lib/inputstreamhelper/widevine/arm.py

```diff
@@ -162,7 +162,8 @@ def install_widevine_arm(backup_path):  # pylint: disable=too-many-statements
         log(4, 'We could not find an ARM device in the Chrome OS recovery.conf')
         ok_dialog(localize(30004), localize(30005))
         return ''
-    required_diskspace = int(arm_device['filesize']) + int(arm_device['zipfilesize'])
+    # Estimated required disk space: take into account up to 5% filesystem overhead
+    required_diskspace = round(1.05 * int(arm_device['zipfilesize']) + int(arm_device['filesize']))
```

Comment:
This would only add 5% of zipfilesize, not 5% of the sum. Is that on purpose?

Reply:
No, I'll add parentheses